tests/utils.go

```diff
@@ -3495,7 +3495,7 @@ func CreateNFSTargetPOD(os string) (nfsTargetIP string) {
 	getStatus := func() k8sv1.PodPhase {
 		pod, err := virtClient.CoreV1().Pods(NamespaceTestDefault).Get(pod.Name, metav1.GetOptions{})
 		Expect(err).ToNot(HaveOccurred())
-		nfsTargetIP = pod.Status.PodIP
+		podIPs = pod.Status.PodIPs
 		return pod.Status.Phase
 	}
 	Eventually(getStatus, 120, 1).Should(Equal(k8sv1.PodRunning))
```

Comment:
You should fetch the NFS target opd object after phase changed to Running to ensure that podIPs is updated 
This will reduces the flakiness I encountered it when working on `CreateISCSITargetPod`

Reply:
`getStatus` is called in the `Eventually` until the status of the pod is `k8sv1.PodRunning` so we are fine. Don't we?

Reply:
I am quite sure there could be a race where `podIPs` is not populated yet and in the same time phase is changed to `Running` and returned (but podIPs is not updated yet)

Reply:
Instead of fetching target pod object after this `Eventualy` can change `getStatus` to
```phase := pod.Status.Phase
podIPs = pod.Status.PodIPs
return phase```